Assets/Src/GameplayView/CellsContent/ContentAudio/CastleAudio/SoundPlayerCastleAudio.cs

```diff
@@ -23,6 +24,14 @@ public override void PlayHitSound()
         public override void PlayDestroySound()
         {
             soundPlayer.PlaySound(_destroySound);
+            Invoke(nameof(FireDestroySoundPlayed), _destroySound.Clip.length);
```

Comment:
Why are we invoking this method in such a way? Mb there's a mismatch , but I couldn't see any parameters in this method's signature too